test/verify/check-system-shutdown-restart

```diff
@@ -44,17 +44,9 @@ class TestShutdownRestart(MachineCase):
         m.execute("useradd user -s /bin/bash -c 'User' || true")
         m.execute("echo user:foobar | chpasswd")
         m.execute("echo 'user ALL=(ALL) ALL' > /etc/sudoers.d/user")
+        self.password = "foobar"
 
-        m.start_cockpit()
-
-        # login
-        b.open("/system")
```

Comment:
Ugh, thanks for the cleanup. I wonder why we did that in the past?

Reply:
It might one of the first tests which just never got ported to the new fangled login_and_go API...